ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -10,13 +10,29 @@ import shared.panels.chat 1.0
 import shared.views.chat 1.0
 import shared.controls.chat 1.0
 
-Column {
+Loader {
     id: root
 
     width: parent.width
     anchors.right: !isCurrentUser ? undefined : parent.right
```

Comment:
Old code but... this double negative, yuck :)

Reply:
You are doubly-right. It is indeed old code. So old that it isn't even needed anymore. Removed.